optaplanner-core/src/test/java/org/optaplanner/core/impl/score/director/drools/testgen/fact/TestGenValueFactTest.java

```diff
@@ -20,14 +20,12 @@
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
-
+import static org.junit.Assert.*;
 import org.junit.Test;
 import org.optaplanner.core.impl.testdata.domain.TestdataEntity;
 import org.optaplanner.core.impl.testdata.domain.TestdataValue;
 import org.optaplanner.core.impl.testdata.domain.collection.TestdataEntityCollectionPropertyEntity;
 
-import static org.junit.Assert.*;
-
```

Comment:
@bradh This is a code style change that is a) not related to the actual fix, and b) doesn't comply with the [code style rules](https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/blob/master/ide-configuration/intellij-configuration/code-style/intellij-code-style_droolsjbpm-java-conventions.xml#L19) set for this project. Please revert it if you are updating this PR. If you plan to contribute on a regular basis, please consider importing the code style rules.